kythe/go/serving/tools/http_server/BUILD

```diff
@@ -20,3 +21,25 @@ go_binary(
         "@org_golang_x_net//http2:go_default_library",
     ],
 )
+
+go_image(
+    name = "http_server_image",
+    srcs = ["http_server.go"],
+    goarch = "amd64",
+    goos = "linux",
+    static = "on",
+    deps = [
+        "//kythe/go/services/filetree",
```

Comment:
Is it possible to not completely mirror the `go_binary` attributes?

Reply:
Based on https://github.com/bazelbuild/rules_docker/blob/master/go/image.bzl#L65, it looks like you can just specify a `binary` attribute.